extensions/dub/src/search-links.tsx

```diff
@@ -9,6 +9,7 @@ import { useState } from "react";
 
 export function SearchLinks() {
   const [query, setQuery] = useState<string | undefined>(undefined);
+  const [showDetails, setShowDetails] = useState<boolean>(false);
```

Comment:
I think we should change this to `useCachedState` so that it remembers user's choice: [example](https://developers.raycast.com/utilities/react-hooks/usecachedstate#example). That should avoid the need of a preference.

```suggestion
  const [showDetails, setShowDetails] = useCachedState<boolean>("show-details", false, {
    cacheNamespace: "dub-search-links"
  });
```